third_party/accessibility/ax/ax_event_generator.cc

```diff
@@ -2,14 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
I accidentally pull the newer version of this file, I intend to leave it as-is. let me know if you think we should go back to the older revision